Libraries/LibWeb/CookieStore/CookieChangeEvent.cpp

```diff
@@ -37,4 +37,12 @@ void CookieChangeEvent::initialize(JS::Realm& realm)
     Base::initialize(realm);
 }
 
+void CookieChangeEvent::visit_edges(Cell::Visitor& visitor)
+{
+    Base::visit_edges(visitor);
+
+    visitor.visit(m_cached_changed);
+    visitor.visit(m_cached_deleted);
```

Comment:
Should we also add a `VISIT_CACHED_ATTRIBUTE` macro? Just seems a little off to have to know the exact name of the cached members here, but nowhere else. No strong feelings.

Reply:
Sure, added, it is a bit nicer